### PR TITLE
[Python] Get table schema

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: python/lancedb/remote/table.py [hunk 1]
```diff
@@ -31,7 +32,12 @@ def __repr__(self) -> str:
         return f"RemoteTable({self._conn.db_name}.{self.name})"
 
     def schema(self) -> pa.Schema:
```
Comment (chebbyChefNEQ) [inline python/lancedb/remote/table.py:34]: let's cache this? we don't need one schema call per table version

File: python/lancedb/remote/db.py [hunk 2]
```diff
@@ -75,4 +76,21 @@ def create_table(
...
+            if schema is None:
+                raise ValueError("Either data or schema must be provided")
+            data = pa.Table.from_pylist([], schema=schema)
+
+        payload = {
+            "name": name,
+            "schema": schema_to_json(data.schema),
+            "records": data.to_pydict(),
+        }
+        self._loop.run_until_complete(self._client.create_table("/table/", payload))
```
Comment (chebbyChefNEQ) [inline python/lancedb/remote/db.py:95]: nit: I think this breaks a notebook. Don't know how to fix it yet. Jupyter will complain about loop already running.

Maybe using a lazy cache_property would fix the problem?

**Review by chebbyChefNEQ — APPROVED** [review]

File: python/lancedb/remote/db.py (hunk 2 above)
Comment (eddyxu) [reply python/lancedb/remote/db.py:95]: Lazy property of the loop object?

File: python/lancedb/remote/table.py (hunk 1 above)
Comment (eddyxu) [reply python/lancedb/remote/table.py:36]: Done

File: python/lancedb/remote/db.py (hunk 2 above)
Comment (chebbyChefNEQ) [reply python/lancedb/remote/db.py:97]: Nvm, just found the solution. https://stackoverflow.com/questions/46827007/runtimeerror-this-event-loop-is-already-running-in-python

**Pull request merged**